### PR TITLE
refactor(core): Move frontend settings to @n8n/api-types (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
## Summary
This is a continuation of #10742 to keep moving stuff out of `n8n-workflow`.

## Review / Merge checklist

- [x] PR title and summary are descriptive

### CODE REVIEW

File: packages/@n8n/api-types/src/settings.ts
Comment (tomi) [inline packages/@n8n/api-types/src/settings.ts:1]: Should this file be named `frontend-settings` to be more explicit?

**Review by tomi — DISMISSED** [review]
Super nice, getting cleaner by the day 🎉 Looks good, one suggestion about file naming

**Review by tomi — APPROVED** [review]
🚀

**Comment by janober** [comment]
Got released with [`n8n@1.60.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.60.0)

**Pull request merged**